logzio-daemonset-rbc.yaml

```diff
@@ -62,10 +62,16 @@ spec:
       - name: fluentd
         image: logzio/logzio-k8s:latest
         env:
-        - name:  LOGZIO_TOKEN
-          value: "your logz.io account token"
-        - name:  LOGZIO_URL
-          value: "your logz.io host url"
+        - name: LOGZIO_TOKEN
```

Comment:
@idohalevi Same note here re: different environment variable names for logs and metrics.